apis/kueue/v1beta1/localqueue_types.go

```diff
@@ -39,8 +39,21 @@ type LocalQueueStatus struct {
 	// admitted to a ClusterQueue and that haven't finished yet.
 	// +optional
 	AdmittedWorkloads int32 `json:"admittedWorkloads"`
+
+	// Conditions hold the latest available observations of the LocalQueue
+	// current state.
+	// +optional
+	// +listType=map
+	// +listMapKey=type
+	Conditions []metav1.Condition `json:"conditions:omitempty"`
 }
 
+const (
+	// ClusterQueueExists indicates clusterQueue that backs this localQueue exists in the cluster and
+	// this localQueue can submit new workloads to its clusterQueue.
+	ClusterQueueExists string = "ClusterQueueExists"
```

Comment:
Maybe the condition should just be `"Active"`, similar to what we have in ClusterQueue. And then when Active=false, the `reason` could be "ClusterQueueDoesNotExist"

Reply:
Sounds good.

Reply:
Done.

Reply:
If the related ClusterQueue exists but is not active, the LocalQueue also can't submit new workloads to its related ClusterQueue, right? In this situation, is the LocalQueue active or not? @alculquicondor

Reply:
IIUC, in that case, the LocalQueue can submit new workloads to related ClusterQueues, although the ClusterQueue can not admit submitted workloads from the LocalQueue.

Reply:
> IIUC, in that case, the LocalQueue can submit new workloads to related ClusterQueues, although the ClusterQueue can not admit submitted workloads from the LocalQueue.

OK. Thanks. It's better to be documented in the user guide. The same keyword `active` in different objects represents different meanings.

Reply:
SGTM.

I created a new issue to keep tracking this: #630

Reply:
> I think we better not make one depend on the other to avoid dependencies between the controllers.

Wait, we don't need to add another notifier. The ClusterQueue notifier is enough. We can just check that the ClusterQueue also has the Active condition.

Reply:
> > I think we better not make one depend on the other to avoid dependencies between the controllers.
> 
> Wait, we don't need to add another notifier. The ClusterQueue notifier is enough. We can just check that the ClusterQueue also has the Active condition.

Does that mean we copy the Active condition of CQ and paste it into the Active condition of Q?

Reply:
If so, if any resourceFlavors are missing, both Active conditions of the CQ and the Q are marked false.
IMO, it sounds a bit weird. 
wdyt?

Reply:
> Does that mean we copy the Active condition of CQ and paste it into the Active condition of Q?

Yeah, that should be enough.

> If so, if any resourceFlavors are missing, both Active conditions of the CQ and the Q are marked false.
IMO, it sounds a bit weird.

How so? Any pending workload in the LocalQueue would remain unschedulable. It's actually better because it's possible that due to RBAC end-users can only see the LocalQueue but not the ClusterQueue.

Reply:
If these active conditions are synchronized, it is difficult for the end user to realize whether the batch admins made a mistake or they (end-users) did.

For example, the controller marks LocalQueue active=false in both cases of the end-users pointing to nonexistent ClusterQueue in LocalQueue and batch Admins forget to deploy some resourceFlavors.

Reply:
Also, it might be better to have ClusterQueueActive=[true|false] conditions in LocalQueueStstus to inform the end users that related CQ is active.

Reply:
You can tell from the Reason. If the ClusterQueue doesn't exist, it will have a different reason than if the ClusterQueue is Inactive.

Reply:
You're right. Sounds good.